tests/complex/test-dynamic-provisioning.sh

```diff
@@ -5,11 +5,6 @@ DOCKER_IMAGE="gcr.io/google_containers/nginx-slim:0.8"
 
 source "${TEST_DIR}/lib.sh"
 
-ssh_config || fail "ERROR creating ssh config"
+pull_docker_image "${DOCKER_IMAGE}"
 
-pull_docker_image "${DOCKER_IMAGE}" || fail "ERROR pulling nginx docker images"
-
-run_on_node "${TEST_DIR}/test-inside-dynamic-provisioning.sh" master || fail
-
-pass
 run -r master "${TEST_DIR}/test-inside-dynamic-provisioning.sh" "Test dynamic provisioning"
```

Comment:
this line must have sneaked into an earlier commit. `run` did not exist before...